labgrid/driver/usbvideodriver.py

```diff
@@ -5,15 +5,16 @@
 from .common import Driver
 from ..factory import target_factory
 from ..exceptions import InvalidConfigError
+from ..protocol import VideoProtocol
 
 @target_factory.reg_driver
 @attr.s(eq=False)
-class USBVideoDriver(Driver):
+class USBVideoDriver(Driver, VideoProtocol):
     bindings = {
         "video": {"USBVideo", "NetworkUSBVideo"},
     }
 
-    def get_caps(self):
+    def get_qualities(self):
```

Comment:
```
Traceback (most recent call last):
  File "/home/phoenix/work/ptx/labgrid/labgrid/remote/client.py", line 1798, in main
    args.func(session)
  File "/home/phoenix/work/ptx/labgrid/labgrid/remote/client.py", line 1161, in video
    drv.stream(quality)
  File "/home/phoenix/work/ptx/labgrid/labgrid/binding.py", line 100, in wrapper
    return func(self, *_args, **_kwargs)
  File "/home/phoenix/work/ptx/labgrid/labgrid/driver/usbvideodriver.py", line 52, in stream
    caps = self.select_caps(caps_hint)
  File "/home/phoenix/work/ptx/labgrid/labgrid/driver/usbvideodriver.py", line 34, in select_caps
    default, variants = self.get_caps()
AttributeError: 'USBVideoDriver' object has no attribute 'get_caps'
```

Reply:
This was resolved by renaming all `get_caps` to `get_qualities`.